### PR TITLE
Monitoring a Microsoft SQL Server database using Solar Winds Database Performance Analyzer - (Duplicate of #3471)

### LINKED ISSUE #2938: Monitoring a Microsoft SQL Server database using Solar Winds Database Performance Analyzer
### Proposed title of article
Database monitoring

### Introduction (2-3 paragraphs)
 This article will explain what database monitoring is and why it should be implemented in the network.
It will also provide insight into what parameters should be monitored for optimum service. The article will aim at enlightening the reader on how to monitor a Microsoft SQL Server database using Solar Winds Database Performance Analyzer (Self-Managed). 

### Key takeaways: 
1. How to deploy the Database performance analyzer in Windows.
2. How to register the database instances for monitoring.
3. Best practice for the installation and configuration process.

### References: 
Not Applicable
### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by ruphus-muita**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #2938 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by ruphus-muita** [comment]
@srishilesh This is the new PR. I believe it is now looking okay

**Comment by srishilesh** [comment]
> @srishilesh This is the new PR. I believe it is now looking okay

@ruphus-muita 

This looks perfect

**Comment by srishilesh** [comment]
Duplicate if #3471

**Comment by WanjaMIKE** [comment]
Could you fix the minor issues highlighted in this plagiarism report. Thanks
[Monitoring a Microsoft SQL Server database using Solar Winds Database Performance Analyzer #3523.pdf](https://github.com/section-engineering-education/engineering-education/files/7072270/Monitoring.a.Microsoft.SQL.Server.database.using.Solar.Winds.Database.Performance.Analyzer.3523.pdf)

**Comment by ruphus-muita** [comment]
> Could you fix the minor issues highlighted in this plagiarism report. Thanks
> [Monitoring a Microsoft SQL Server database using Solar Winds Database Performance Analyzer #3523.pdf](https://github.com/section-engineering-education/engineering-education/files/7072270/Monitoring.a.Microsoft.SQL.Server.database.using.Solar.Winds.Database.Performance.Analyzer.3523.pdf)

@WanjaMIKE  This done. Please check. Thanks

**Comment by srishilesh** [comment]
@ruphus-muita 

Thanks for the change.

I'll complete my next round of review by tomorrow.

**Comment by ruphus-muita** [comment]
> @ruphus-muita 
> 
> Thanks for the change.
> 
> I'll complete my next round of review by tomorrow.

Are you done?

File: content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+### Introduction
+Databases hold a lot of critical and highly sensitive data for various organizations. For this reason, it is crucial to monitor them and hence make sure they are securely accessed. The article will aim at enlightening the reader on how to monitor a Microsoft SQL Server database using Solar Winds Database Performance Analyzer (Self-Managed). This is a software offered by Solar Winds and is very useful in attaining the security of a database.
+
```
Comment (srishilesh) [inline content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md:3]: Can you talk more about the goals of this tutorial? What are the key takeaways? Objective of the code tutorial?

File: content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+### Prerequisites
+1.	Correct and needed credentials
+2.	Correct system requirements
+3.	A powerful server where the DPA (Database performance analyzer) will be installed.
+4.	A repository database
+5.	Customer portal account to help access licenses and downloads. The customer portal can be accessed using this [link](https://customerportal.solarwinds.com/).
+6.	A local administrator account to be used by the server hosting DPA.
+7.	Updated operating system.
+8.	Open the following ports on the server and firewall: 8124, 8123, 8127, 443, 80, and 587.
+9.	Exclude the DPA directory from antivirus scanning to ensure smooth installation: C:\Program Files\SolarWinds\DPA
```
Comment (srishilesh) [inline content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md:21]: These need not be a numbered list - convert them to normal bulleted points `-`

File: content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+Note: DPA is short for Database Performance Analyzer.
+
+### Phase 1
+
+### Installing the DPA on windows
+1.	If you're installing a trial version, [click here](https://www.solarwinds.com/database-performance-analyzer) or the SolarWinds Customer Portal to get the installation file.
+2.	Log in to the Windows server using the administrator account.
+3.	Extract the ZIP file you downloaded into a traceable folder.
+
+![File Extraction](/engineering-education/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/File Extraction.png)
```
Comment (srishilesh) [inline content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md:32]: I would suggest using image names separated by `-`, and all the letters in small case.

Comment (srishilesh) [reply content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md:32]: Could you do this change for all the images?

File: content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+4.	Open SolarWindsDPASetup-x64.exe and run it. Click next once the installer has started.
```
Comment (srishilesh) [inline content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md:34]:
```suggestion
4.	Open `SolarWindsDPASetup-x64.exe` and run it. Click next once the installer has started.
```

File: content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+![Success-Message](/engineering-education/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/Success-Message.png)
+
+### Phase 3
+
+### How to register the database instances for monitoring.
+Identifying the privileged user
+	The privileged can create a monitoring user or assigns the needed rights to an existing user who now becomes a monitoring user.
+	The privileged account can create a monitoring user or assigns the needed rights to an existing user.
+	Note: The privileged user's credentials are not stored by the DPA.
+	The privileged user must have the following privileges for self-managed SQL Server database instances: SYSADMIN (system administrator).
```
Comment (srishilesh) [inline content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md:96]: Can you converted these bullets to `-`?

File: content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+![SQL Self-mananged](/engineering-education/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/SQL Self-mananged.png)
+
+#### Stage 3 : Input connection information
+1. Select Next and finish the other steps as described below:
+2. Input the connection information for the Monitored Database Instance enter the following connection information.
+3. When service for SQL Server Browser is available, enter the IP address or hostname as well as name of the instance, in the following format: Server\Instance.
+4. See the [Note](Note-:-Each-instance-must-be-registered-separately) below for a guide to register replicas if the registered instance has additional groups.  
+5. Otherwise, input the name of the server or its IP address, as well as the port number, if necessary.
+
+#### Note: Each instance must be registered separately
```
Comment (srishilesh) [inline content/articles/monitoring-microsoft-sql-server-using-solar-windows-database-analyzer/index.md:115]:
```suggestion
> NOTE: Each instance must be registered separately
```

**Review by srishilesh — CHANGES_REQUESTED** [review]
@ruphus-muita 

I've suggested a few changes. Please ping me when done.

I would suggest you to rename the images - all in small case, separated by `-`.

On completion, we'll move to the tech review.

Thanks.

**Comment by ruphus-muita** [comment]
@srishilesh All done

**Comment by ruphus-muita** [comment]
@srishilesh are we done with this PR?

**Comment by srishilesh** [comment]
> @srishilesh are we done with this PR?

@ruphus-muita 

The final round of review by me, is in progress. Please wait patiently.

**Review by srishilesh — APPROVED** [review]
@ruphus-muita 

Good job.

I've made a few changes to your article. Make sure to go through the commits to understand them

Moving for the final review.

Thanks.

**Comment by WanjaMIKE** [comment]
@ruphus-muita could you perform more analyses using the tool. I found that most of the article focused on the installation part which seems more straight-forward. Thanks

**Comment by ruphus-muita** [comment]
@WanjaMIKE Kindly clarify on what you mean by more analyses.

**Comment by WanjaMIKE** [comment]
I mean use the tool to conduct further tests and then explain them in the article. Thanks

**Comment by ruphus-muita** [comment]
> I mean use the tool to conduct further tests and then explain them in the article. Thanks

@WanjaMIKE This is done. Kindly confirm

**Comment by hectorkambow** [comment]
Overall nice job gents 👍

**Pull request merged**